proxysql-status

```diff
@@ -170,11 +175,17 @@ fi
 
 if [[ $DUMP_ALL -eq 1 || $DUMP_STATS -eq 1 ]]; then
     echo "............ DUMPING STATS DATABASE ............"
-    TABLES=$(mysql_exec -BN "SHOW TABLES FROM stats" 2> /dev/null)
+    TABLES=$(mysql_exec -BN "SHOW TABLES FROM stats" 2> /dev/null)      
     for table in $TABLES
     do
         if [[ -n $TABLE_FILTER && $table != *${TABLE_FILTER}* ]]; then
             continue
+        fi
+		# Dump _reset tables only if we specify option --with-stats-reset
+        if [[ $DUMP_STATS_RESET_TABLE -eq 0 ]]; then
```

Comment:
The --with-stats-reset flag sets DUMP_STATS_RESET_TABLE to 1 not 0, should this test be
    if [[ $DUMP_STATS_RESET_TABLE -eq 1 ]]; then
or is the comment wrong?

Reply:
By default, proxysql-status dump all tables. To avoid that we had given this condition like that to skip the _reset tables.